src/controllers/developers.js

```diff
@@ -62,3 +58,29 @@ export const postDeveloper = async (req, res, next) => {
     return next(err);
   }
 };
+
+export const getDeveloper = async (req, res, next) => {
+  const connection = createDbConnection();
+  const Developer = connection.model('Developer', developerSchema);
+  const { id } = req.params;
+  try {
+    const developer = await Developer.find({ _id: id });
+
+    if (!developer.length) {
+      throw new Error("Developer doesn't exist");
```

Comment:
nit: please use single quotes
```suggestion
      throw new Error('Developer doesn\'t exist');
```